powerquery-docs/query-folding-basics.md

```diff
@@ -9,22 +9,17 @@ ms.topic: overview
 
 # Overview of query evaluation and query folding in Power Query
 
-This article provides a basic overview of how M queries are processed and turned into data source requests.
+This article provides an overview of how M queries are processed and turned into data source requests.
 
 ## Power Query M script
 
-Any query, whether created by Power Query, manually written by you in the advanced editor, or entered using a blank document, consists of functions and syntax from the [Power Query M formula language](/powerquery-m/). This query gets interpreted and evaluated by the Power Query engine to output its results. The M script serves as the set of instructions needed to evaluate the query.
+Any query, whether created by Power Query, manually written in the advanced editor, or entered using a blank document, consists of functions and syntax from the [Power Query M formula language](/powerquery-m/). M script, also referred to as M code, is any code that uses the M language inside a Power Query query. While we recommend that you use the Power Query editor to create most of the M script, you can manually modify your M script. 
 
->[!TIP]
->You can think of the M script as a recipe that describes how to prepare your data.
+The M script is accessible through the advanced editor window or the script view in the formula bar. M script serves as the set of instructions needed by the Power Query engine to evaluate queries and output results. You can think of the M script as a recipe that describes how to prepare your data.
 
-The most common way to create an M script is by using the Power Query editor. For example, when you connect to a data source, such as a SQL Server database, you'll notice on the right-hand side of your screen that there's a section called [applied steps](applied-steps.md). This section displays all the steps or transforms used in your query. In this sense, the Power Query editor serves as an interface to help you create the appropriate M script for the transforms that you're after, and ensures that the code you use is valid.
+## Applied steps 
 
->[!NOTE]
-> The M script is used in the Power Query editor to:
->
-> - Display the query as a series of steps and allow the creation or modification of new steps.
-> - Display a diagram view.
```

Comment:
I actually do not understand this note. If this is important it should go to a normal paragraph